acquia-http-hmac.gemspec

```diff
@@ -13,9 +13,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.executables << 'acq-http-request'
 
-  s.add_runtime_dependency('excon', '~> 0.44', '>= 0.44.4')
```

Comment:
This was here since I was starting on a client using excon.  But An initial client could just use stdlib or we could make excon optional